.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: CI on push release tag
+on:
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  run-tests:
+    name: Run DTS tests
+    if: ${{ github.head_ref == 'develop' }}
+    runs-on:
+      labels: dts-builder
+    steps:
+      - name: Checkout OSFV repo
+        shell: bash
+        run: |
+          git clone https://github.com/Dasharo/open-source-firmware-validation.git
+      - name: Install requirements
+        shell: bash
+        run: |
+          cd open-source-firmware-validation
+          git submodule update --init --checkout
+          python3 -m virtualenv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+      - name: Run QEMU
+        shell: bash
+        id: run_qemu
+        run: |
+          ./scripts/ci/qemu-run.sh nographic firmware &
+          pid=$!
+          echo "qemu_pid=$pid" >> "$GITHUB_OUTPUT"
+      # TODO: Add DPP keys to secrets and use them in tests
+      - name: Run tests
+        shell: bash
+        env:
+          DPP_EMAIL: ${{secrets.DPP_EMAIL}}
+          DPP_PASSWORD: ${{secrets.DPP_PASSWORD}}
+        run: |
+          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
+             dts/dts-e2e.robot > /tmp/dts-test-ci
+      - name: Cleanup
```

Comment:
Add `if: always()` so it always runs?
https://github.com/Dasharo/meta-dts/blob/6818b595c8a7f0930d693f8bdf9091c8833bd90b/.github/workflows/ci.yml#L144

Reply:
Here I'm using this as a step, not as a separate job. Makes it easier to pass `qemu_pid`.

Reply:
https://github.com/Dasharo/meta-dts/compare/df52aad37e5be1793811bae2603d8c3daec0688f..21a11dd5a691bfea6616bb097deeb37071100320

Reply:
isn't `if` reserved for jobs, not steps?

Reply:
Will `if always()` work here?

Reply:
I'm not sure, needs to be tested, based on https://stackoverflow.com/a/58859404 yes, it will always run, even if job is cancelled.

Reply:
https://github.com/Dasharo/meta-dts/compare/21a11dd5a691bfea6616bb097deeb37071100320..878c79a1b298c6c878094a004991c25761354cfa